src/Microsoft.AspNet.Mvc.Core/Controller.cs

```diff
@@ -400,6 +401,28 @@ public virtual JsonResult Json(object data)
         }
 
         /// <summary>
+        /// Creates a <see cref="JsonResult"/> object that serializes the specified <paramref name="data"/> object
+        /// to JSON.
+        /// </summary>
+        /// <param name="data">The object to serialize.</param>
+        /// <param name="serializerSettings">The <see cref="JsonSerializerSettings"/> to be used by the formatter.</param>
+        /// <returns>The created <see cref="JsonResult"/> that serializes the specified <paramref name="data"/>
+        /// to JSON format for the response.</returns>
+        /// <remarks>It is recommended to cache the <see cref="JsonSerializerSettings"/> instance for the action
```

Comment:
`serializes the specified <paramref name="data"/> object to JSON.` -> `as JSON.`

`It is recommended to cache the <see cref="JsonSerializerSettings"/> instance for the action...` -> 

`Callers should cache an instance of <see cref="JsonSerializerSettings"/> to avoid recreating cached data with each call`.